co-noir/co-builder/src/types/mod.rs

```diff
@@ -0,0 +1,4 @@
+#[allow(unused)] // TACEO TODO remove later
```

Comment:
do we keep this around?

Reply:
Until more features of the noir VM are implemented yes